### PR TITLE
Add ISO20022 to SwiftMT 1XX DataMappers

### PR DESCRIPTION
**Pull request by KATTA-00**
Added mappers for the following ISO20022 MX to Swift MT message types:

- Pain.008 → MT104
- Pain.001 → MT101
- Pacs.008 → MT102, MT102 STP, MT103, MT103 STP, MT103 REMIT
- Pacs.003 → MT104, MT107
- Camt.056 → MT192
- Camt.055 → MT192
- Camt.031 → MT196
- Camt.028 → MT196
- Camt.026 → MT195

### CODE REVIEW

File: mapper/.devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (RivinduM) [inline mapper/.devcontainer.json:1]: Let's change the directory name to `iso20022_to_switmt` to make it consistent with the other repo

File: mapper/.devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "image": "ballerina/ballerina-devcontainer:2201.10.0",
+  "customizations": {
```
Comment (RivinduM) [inline mapper/.devcontainer.json:3]: Do we need to add this [.devcontainer.json](https://github.com/ballerina-platform/module-ballerinax-financial.iso20022-to-swiftmt/pull/3/files#diff-c140fdb96d32e8a31526c5ae262ac369ad893354744905009d661c52c580a8ce) file?

File: mapper/camt_026_transform.bal [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import ballerinax/financial.swift.mt as swiftmt;
+
+# Transforms a camt.026 ISO 20022 document to its corresponding SWIFT MT195 format.
+#
+# + document - The camt.026 document to be transformed.
+# + return - The transformed SWIFT MT195 message or an error.
+isolated function transformCamt026ToMT195(camtIsoRecord:Camt026Document document) returns swiftmt:MTn95Message|error => let
+    camtIsoRecord:SupplementaryData1[]? splmtryData = document.UblToApply.SplmtryData
+    in {
+        block1: check generateBlock1FromAssgnmt(document.UblToApply.Assgnmt),
```
Comment (RivinduM) [inline mapper/camt_026_transform.bal:27]:
```suggestion
        block1: check generateMtBlock1FromAssgnmt(document.UblToApply.Assgnmt),
```
keep the function names consistent

File: mapper/mt101_utilities.bal [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    painIsoRecord:OrganisationIdentification39? OrgId = id?.OrgId;
+    if (OrgId != ()) {
+        return <swiftmt:MT50C>{
+            name: MT50C_NAME,
+            IdnCd: {
+                content: getEmptyStrIfNull(OrgId?.AnyBIC),
+                number: NUMBER1
+            }
+        };
+    }
```
Comment (RivinduM) [inline mapper/mt101_utilities.bal:41]: Do we need seperate utility classes. Can't we make them common?

File: mapper/tests/test.bal [hunk 5]
```diff
@@ -0,0 +1,736 @@
...
+""}FINPETROL INC.</Nm><PstlAdr><TwnNm>HELSINKI</TwnNm><Ctry>FI</Ctry><AdrLine>ANDRELAE SPINKATU 7</AdrLine></PstlAdr><Id><OrgId/></Id></Cdtr><CdtrAcct><Id><Othr><Id>${
+""}9020123100</Id><SchmeNm><Cd>BBAN</Cd></SchmeNm></Othr></Id></CdtrAcct><InstrForCdtrAgt/><InstrForDbtrAgt><Cd>${
+""}CMZB</Cd></InstrForDbtrAgt><RmtInf><Ustrd/></RmtInf></CdtTrfTxInf></PmtInf></CstmrCdtTrfInitn></Document>`;
+
+    painIsoRecord:Pain001Document pain001Message = check xmldata:parseAsType(documentXML);
+    swiftmt:MT101Message|error mt101Message = transformPain001DocumentToMT101(pain001Message);
+
+    if (mt101Message is swiftmt:MT101Message) {
+        test:assertEquals(mt101Message.block2.messageType, "101");
+        test:assertEquals(pain001Message.CstmrCdtTrfInitn.GrpHdr.MsgId, mt101Message.block4.MT21R?.Ref?.content, "Message ID mismatch");
```
Comment (RivinduM) [inline mapper/tests/test.bal:66]: Let's add a validation to assert the fin message as suggested during the code review

File: mapper/utility.bal [hunk 6]
```diff
@@ -0,0 +1,1229 @@
...
+// under the License.
+
+import ballerina/regex;
+import ballerina/time;
+import ballerinax/financial.iso20022.cash_management as camtIsoRecord;
+import ballerinax/financial.iso20022.payment_initiation as painIsoRecord;
+import ballerinax/financial.iso20022.payments_clearing_and_settlement as pacsIsoRecord;
+import ballerinax/financial.swift.mt as swiftmt;
+
+// TODO: Add the necessary functions to map the MX messages to the MT messages.
```
Comment (RivinduM) [inline mapper/utility.bal:24]: What's the missing part here? Can u elaborate more on the Todos. Check other places as well

File: mapper/.devcontainer.json (hunk 1 above)
Comment (KATTA-00) [reply mapper/.devcontainer.json]: Ack

File: mapper/.devcontainer.json (hunk 2 above)
Comment (KATTA-00) [reply mapper/.devcontainer.json]: It was generated when project init, Lets remove it

File: mapper/camt_026_transform.bal (hunk 3 above)
Comment (KATTA-00) [reply mapper/camt_026_transform.bal]: Ack

File: mapper/mt101_utilities.bal (hunk 4 above)
Comment (KATTA-00) [reply mapper/mt101_utilities.bal]: For ISO documents, they can have different record types. Thus so we can't use common methods for some mappings.

File: mapper/tests/test.bal (hunk 5 above)
Comment (KATTA-00) [reply mapper/tests/test.bal]: Ack

**Pull request merged**